### PR TITLE
Adding e3sm-kernels to Spack

### PR DESCRIPTION
**Pull request by rspavel**
Added e3sm-kernels spackage

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed sha256 and branch exists.

File: var/spack/repos/builtin/packages/e3sm-kernels/package.py
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/e3sm-kernels/package.py:21]: Neither of these accounts seem to be accessible here, at least I can't include them with `@<account>`. Would you care to be the maintainer *of this package*?

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/e3sm-kernels/package.py:21]: Hmm.  I wonder why `spackbot` can ping them but I can't.

Comment (rspavel) [reply var/spack/repos/builtin/packages/e3sm-kernels/package.py:21]: I grabbed them from the actual repo of the code itself. Let me ping a few other people to see if they are available before we fall back to that. Looks like I failed to run `spack style` correctly anyway and need to fix an import order

**Comment by tldahlgren** [comment]
Please fix the import order issue reported by the style checks.

File: var/spack/repos/builtin/packages/e3sm-kernels/package.py
Comment (philipwjones) [reply var/spack/repos/builtin/packages/e3sm-kernels/package.py:21]: Ummm...not sure what this is about? @sarats - do you know anything?

Comment (rspavel) [reply var/spack/repos/builtin/packages/e3sm-kernels/package.py:21]: The ECP Proxy Apps project are cleaning up and adding a few proxy apps to spack that were previously part of the catalog.

I forget what the official responsibility of a maintainer in spack is but mostly it boils down to taking a look at what people do and making sure it isn't too insane or wrong.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/e3sm-kernels/package.py:21]: That's correct.  Signing up to be a `maintainer` essentially means you will get "pinged" when someone submits a PR to change the package to give you a chance to review the changes.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/e3sm-kernels/package.py:21]: @philipwjones @sarats Let us know if you do not want to be included in the `maintainers` list so it can be updated before this PR is merged.

**Comment by tldahlgren** [comment]
I'm going to merge.  If @philipwjones and or @sarats want to be removed from the list (for notifications) that can be done in a separate PR.

**Review by tldahlgren — APPROVED** [review]

**Comment by sarats** [comment]
This is picking up just a couple of small kernels from E3SM. 

Anyway, leave me there for now. We will revisit as needed.

**Pull request merged**
